### ISSUE TITLE
`ts/lines-between-class-members`: does not apply to interface members

### ISSUE THREAD

**Issue by ygrandgirard** [eslint-stylistic/eslint-stylistic#353]
### Validations

- [X] Follow our [Code of Conduct](https://github.com/eslint-community/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

The `ts/lines-between-class-members` rule does not apply to interface members.

Given this config:
```json
{
  "rules": {
    "@stylistic/ts/lines-between-class-members": "error"
  }
}
```

This will be reported:
```ts
class A {
  a: string;
  b: string; // Expected blank line between class members.
}
```

But not this:
```ts
interface A {
  a: string;
  b: string;
}
```

### Reproduction

https://github.com/eslint-community/eslint-stylistic/pull/352

### Contributes

- [X] I am willing to submit a PR to fix this issue
- [X] I am willing to submit a PR with failing tests
